example/Clarinet.toml

```diff
@@ -26,8 +34,8 @@ path = "contracts/slice.clar"
 clarity_version = 3
 epoch = 3.0
 
-[contracts.trait]
-path = 'contracts/trait.clar'
+[contracts.ft-transfer-many]
+path = 'contracts/ft-transfer-many.clar'
```

Comment:
Some paths in this file use double quotes and some others use single quotes. Let's stick with one?